code/src/main/orxAndroidNativeSupport.cpp

```diff
@@ -68,6 +80,9 @@ typedef struct __orxANDROID_STATIC_t {
         struct android_app* app_;
 } orxANDROID_STATIC;
 
+// Guard against too early access of sstAndroid which is initialized in new thread created in native glue.
+static orxBOOL sstAndroidInitialized = orxFALSE;

```

Comment:
this doesn't seems to be used anymore, can be removed

Reply:
Removed now